test/test-lint.ts

```diff
@@ -34,36 +35,18 @@ const OPTIONS: Options = {
 const BAD_CODE = `throw 'hello world';`;
 const GOOD_CODE = `throw new Error('hello world');`;
 
-const TSLINT_CONFIG = require('../../tslint.json');
-
 test.serial('createProgram should return an object', async t => {
   await withFixtures({'tsconfig.json': '{}'}, async () => {
     const program = lint.createProgram(OPTIONS);
     t.truthy(program);
   });
 });
 
-// TODO: make the error friendlier. This error implies that our module messed
```

Comment:
Why is this test deletec?

Reply:
The tests in this fine weren't fully thought through, and this one in particular wasn't needed. Instead of fixing the bug in `lint.ts`, the tests were always ensuring that there was a `tslint.json` in the specific test directory. This test was testing that the bug correctly existed 🤣.

Reply:
Sorry, this should had said.. fix the bug in the `OPTIONS` object construction.